content/04-distribute/distribute-sdk.mdx

```diff
@@ -40,34 +50,46 @@ We assume that:
     <MultiCodeBlock>
 
     ```typescript
-    const token: SecurityToken = await apiCeo.getSecurityToken({
+    const asset: Asset = await apiCeo.assets.getAsset({
         "ticker" :"ACME"
     });
     ```
 
     ```javascript
-    const token = await apiCeo.getSecurityToken({
+    const asset = await apiCeo.assets.getAsset({
         "ticker" :"ACME"
     });
     ```
 
     </MultiCodeBlock>
 * `acme` is the owner of the ACME token. `acme` will also be the token's primary issuance agent;
 * Alice has an account, `alice`, that represents her as an individual;
-* Alice the individual also can [instantiate](https://developers.polymath.network/polymesh-sdk-api-reference/classes/polymesh#static-connect) a Polymesh client whenever she needs to act with:
+* Alice the individual also can [instantiate](https://github.com/PolymathNetwork/polymesh-sdk/wiki/api.client.Polymesh.Polymesh#connect) a Polymesh client whenever she needs to act with:
     <MultiCodeBlock>
 
     ```typescript
+    const signingManagerAlice: LocalSigningManager = await LocalSigningManager.create({
+        "accounts": [{
+            "mnemonic": "word1 word2 ..."
+        }]
+    });
+
     const apiAlice: Polymesh = await Polymesh.connect({
-        "nodeUrl": "wss://testnet-rpc.polymesh.live",
-        "accountMnemonic": "word1 word2 ..."
+        "nodeUrl": "wss://testnet-rpc.polymesh.live", // For instance, or the proper network
```

Comment:
```suggestion
        "nodeUrl": "wss://testnet-rpc.polymesh.live", // or your preferred node
```

Its not just the network, but what node you want to trust. There are many valid values for any network